charts/dawarich/values.yaml

```diff
@@ -2,6 +2,22 @@ config:
   timezone: Europe/Amsterdam
   minimumMinutesSpentInCity: 60
   jobConcurrency: 10
+  oidc:
+    enabled: false
+    issuerUrl: https://example.com/
+    clientId: dawarich
+    clientSecret: your-client-secret
+    redirectUri: https://timeline.example.com/users/auth/openid_connect/callback
+    secretName: ""
+    # Optional, override the default references to the keys in the secret above
+    secretKeyRef:
+      issuerUrl: issuerUrl
+      clientId: clientId
+      clientSecret: clientSecret
+      redirectUri: redirectUri
+    autoRegister: false
+    providerName: YourProviderName
+    allowEmailPasswordRegistration: false
```

Comment:
This is theoretically not related to oidc, this should not be under the oidc key